react_native_fundamentals/index.md

```diff
@@ -1,7 +1,23 @@
 # Module content
 
-- ToDo
-- ToDo
-- ToDo
-- ToDo
-- ToDo
+1. Introduction 
+    - iOS and Android overview
+    - Types of apps
+    - What is React Native
+    - Expo vs React Native
+    - Alternatives 
+2. Environment setup
+3. Creating a new app
+4. Project structure overview
+5. Core Components
+6. Styles
+    - StyleSheet 
+    - Flexbox
+7. List Views 
```

Comment:
I propose to rename this item to Handling interactions and apart from list views discuss buttons and touchables here as well.